app/controllers/vatFinancials/vatAccountingPeriod/AccountingPeriodController.scala

```diff
@@ -44,15 +40,12 @@ class AccountingPeriodController @Inject()(ds: CommonPlayDependencies)
     form.bindFromRequest().fold(
       badForm => BadRequest(views.html.pages.vatFinancials.vatAccountingPeriod.accounting_period(badForm)).pure,
       data => for {
-        originalTurnover <- keystoreConnector.fetchAndGet[Long](lastKnownValueKey)
         _ <- save(data)
-        _ <- vrs.submitVatFinancials()
-        turnover <- viewModel[EstimateVatTurnover]().fold[Long](0)(_.vatTurnoverEstimate)
-        _ <- vrs.conditionalDeleteElement(VatFlatRateSchemePath, originalTurnover.getOrElse(0) != turnover)
-      } yield Redirect(if (turnover > joinThreshold) {
-        controllers.routes.SummaryController.show()
+        voluntaryReg <- viewModel[VoluntaryRegistration]().fold(true)(_ == VoluntaryRegistration.yes)
```

Comment:
i remember we were discussing if defaulting to true here is a valid thing to do...